BootCampV2/OtherStuff/.gitignore

```diff
@@ -0,0 +1,350 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
https://stackoverflow.com/questions/348170/how-do-i-undo-git-add-before-commit
Please don't include a million files in one PR. If you accidentally committed something- delete, commit and push again.
If you don't want trash files added, yet you have already committed them- you can undo it by:
https://stackoverflow.com/questions/348170/how-do-i-undo-git-add-before-commit